cmd/cli/plugin/package/kctrl/README.md

```diff
@@ -0,0 +1,534 @@
+# Package
+
+Manage package lifecycle operations.
+
+## Usage
+
+Package and Repository Operations ( Subject to Change ):
+
+```sh
+>>> tanzu package --help
+Tanzu package management (available, install, installed, repository)
+
+Usage:
+  tanzu package [flags]
+  tanzu package [command]
+
+Available Commands:
+  available     Manage available packages (get, list)
+  completion    Generate the autocompletion script for the specified shell
+  install       Install package
+  installed     Manage installed packages (create, delete, get, list, update)
+  repository    Manage package repositories (add, delete, get, list, update)
+
```

Comment:
Wondering if the Readme content should be replaced or not considering that kctrl package plugin is gonna be behind feature flag for now.

Reply:
> kctrl package plugin is gonna be behind feature flag for now.

Yep, considering that, I have not updated the existing README, I have created a new one in the kctrl package. We can replace the existing README with the new one when we decide to make this as the default behaviour.